src/cli/util.js

```diff
@@ -443,7 +443,13 @@ function formatFiles(context) {
   }
 
   eachFilename(context, context.filePatterns, filename => {
-    const fileIgnored = ignorer.filter([filename]).length === 0;
+    let fileIgnored = false;
+    try {
+      fileIgnored = ignorer.ignores(path.relative(process.cwd(), filename));
+    } catch (_) {
+      // eslint-disable-line no-empty
```

Comment:
You can change this to `// Do nothing`. A comment inside a block counts as non-empty in ESLint. This way it’s explicit that you actually intended to ignore the error.

Reply:
Btw, why _are_ we ignoring errors here?

Reply:
https://github.com/kaelzhang/node-ignore#upgrade-4x---5x

`ignorer.ignores('../file-from-parent')`

throws

Reply:
I did tried `isValidPath`, but they are the same.

Reply:
> You can change this to `// Do nothing`. A comment inside a block counts as non-empty in ESLint. This way it’s explicit that you actually intended to ignore the error.

thanks